prboom2/src/dsda/stretch.c

```diff
@@ -293,6 +299,14 @@ void dsda_EvaluatePatchScale(void) {
       break;
   }
 
+  // Raven HUD breaks in "patch_stretch_not_adjusted",
+  // so let's use "patch_stretch_doom_format" settings
+  if (raven && render_stretch_hud == 0) {
+      ST_SCALED_HEIGHT = g_st_height * WIDE_SCREENHEIGHT / 200;
+      wide_offset2x = SCREENWIDTH - WIDE_SCREENWIDTH;
+      wide_offset2y = SCREENHEIGHT - WIDE_SCREENHEIGHT;
+  }
+
```

Comment:
Why did you make this change? It does the exact same thing
Althought instead of `render_stretch_hud++`, you could do `render_stretch_hud = patch_stretch_doom_format`

Reply:
Original code changes every game asset that's "not adjusted" to become "doom format".

My code only does that to the statusbar as the statusbar doesn't display properly in "not adjusted".

But my code allows all other assets to be able to render as "not adjusted", which previously was not the case.

Reply:
Oh I see. The old code permanently changes the setting.
Missed that 😅

Reply:
But instead of doing `== 0`, do `== patch_stretch_not_adjusted`for readability